docs/make.jl

```diff
@@ -160,9 +160,9 @@ makedocs(bib,
  checkdocs = :exports
 )
 
-@info "Cleaning up temporary .jld2 and .nc files created by doctests..."
+@info "Cleaning up temporary .jld2 and .nc output created by doctests or literated examples..."
 
-for file in vcat(glob("docs/*.jld2"), glob("docs/*.nc"))
+for file in vcat(glob("docs/**/*.jld2"), glob("docs/**/*.nc"))
```

Comment:
Maybe

```suggestion
for file in vcat(glob("docs/**/*.jld2"), glob("docs/**/*.nc"), glob("docs/*.jld2"), glob("docs/*.nc"))
```

?

Reply:
Hm... then we need to figure out a way to include all subdirectories recursively

Reply:
for now I added the directories that output lives in... It's not the best solution but it's better than uploading `.nc` files in the Docs repository!!